index.js.flow

```diff
@@ -1,5 +1,5 @@
-interface ... {
-  // ...
-}
-
-declare export default function use...(): ...;
+declare export default function useOnlineStatus(opts: {
+  onOnline?: () => void,
+  onOffline?: () => void,
+  onStatusChange?: (online: boolean) => void,
+}): boolean;
```

Comment:
I'm not familiar to Flowtype, but could these typings be extracted to a separate type, like my suggestion on the TypeScript PR?

Something like:

```tsx
interface OnlineStatusOpts {
    onOnline?: () => void,
    onOffline?: () => void,
    onStatusChange?: (online: boolean) => void,
}

export default function useOnlineStatus(opts: OnlineStatusOpts): boolean;
```